app/dao/services_dao.py

```diff
@@ -324,14 +324,19 @@ def dao_fetch_todays_stats_for_all_services(include_from_test_key=True):
 
 @statsd(namespace='dao')
 def fetch_stats_by_date_range_for_all_services(start_date, end_date, include_from_test_key=True):
+    if not isinstance(end_date, datetime):
```

Comment:
can we move the call of `get_london_midnight_in_utc` into this function, so this function is only ever called with naive date objects, which i feel makes most logical sense